packages/terra-clinical-onset-picker/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "terra-clinical-onset-picker",
+  "version": "0.0.0",
+  "description": "The terra-clinical-onset-picker component provides users a way to enter or select an approximate date for use in onset scenarios.",
+  "main": "lib/OnsetPicker.js",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-clinical.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "Clinical",
+    "Form",
+    "UI",
+    "terra-clinical-onset-picker",
+    "Date",
+    "onset"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-clinical/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-clinical#readme",
+  "devDependencies": {
+    "react-intl": "^2.3.0",
+    "terra-props-table": "^1.3.0"
+  },
+  "peerDependencies": {
+    "react": "^15.4.2",
+    "react-dom": "^15.4.2",
+    "terra-base": "^2.0.0"
+  },
+  "dependencies": {
+    "moment": "^2.17.1",
+    "prop-types": "^15.5.8",
+    "terra-base": "^1.3.0",
+    "terra-date-picker": "^1.9.0",
+    "terra-i18n": "^1.4.0",
+    "terra-form": "^1.3.0"
+  },
+  "scripts": {
+    "compile": "npm run compile:clean && npm run compile:build",
+    "compile:clean": "$(cd ..; npm bin)/rimraf lib",
+    "compile:build": "$(cd ..; npm bin)/babel src --out-dir lib --copy-files",
+    "lint": "npm run lint:js",
+    "lint:js": "$(cd ..; npm bin)/eslint --ext .js,.jsx . --ignore-path ../../.eslintignore",
+    "props-table": "$(cd ..; npm bin)/props-table ./src/OnsetPicker.jsx --out-dir ./docs/props-table",
+    "test": "npm run test:jest && npm run test:nightwatch",
+    "test:jest": "$(cd ..; npm bin)/jest --config ../../jestconfig.json",
+    "test:nightwatch": "$(cd ..; npm bin)/nightwatch -c ../../nightwatch.conf.js"
```

Comment:
The `$(cd ..; npm bin)/` can be dropped. See [here](https://github.com/cerner/terra-clinical/blob/master/packages/terra-clinical-error-view/package.json#L39-L48) for example.

Reply:
Removed